adoc/chapters/acknowledgements.adoc

```diff
@@ -90,6 +90,7 @@
   * Peter Thoman, University of Innsbruck
   * Biagio Cosenza, University of Salerno
   * Paul Preney, University of Windsor
+  * Atharva Dubey, Codeplay
```

Comment:
The list is grouped by organization and then sorted by last name within each organization.  Can you move your name to appropriate place in the Codeplay section?

Reply:
Thanks for letting me know, will do.

Reply:
Done